dotnet/Services/ProductReviewService.cs

```diff
@@ -333,8 +333,34 @@ public async Task<ReviewsResponseWrapper> GetReviewsByProductId(string productId
             {
                 searchQuery = $"{searchQuery}&approved=true";
             }
-            
-            ReviewsResponseWrapper wrapper = await this._productReviewRepository.GetProductReviewsMD($"productId={productId}{sort}{searchQuery}{ratingQuery}", from.ToString(), to.ToString());
+
+            if (pastReviews)
+            {
+                string productQuery = $" AND productId={productId}";
+                if (rating > 0 && rating <= 5)
+                {
+                    ratingQuery = $" AND rating={rating}";
+                }
+                if (!string.IsNullOrEmpty(locale))
+                {
+                    localeQuery = $"((locale=*{locale}-*) OR (locale is null))";
```

Comment:
Is the dash after locale correct?

Reply:
The locales values are like `en-US`, I think the search will work with or without the dash. Do you think it would be better without the dash?